requirements-test.txt

```
@@ -12,8 +12,8 @@ pandas==2.1.3 ; python_version < '3.11'
 pandas==2.2.3 ; python_version >= '3.11'
 xgboost==2.1.4 ; python_version < '3.10'
 xgboost==3.0.0 ; python_version >= '3.10'
-lightgbm==4.6.0
-catboost==1.2.8 ; python_version < '3.11' # TODO: Remove 3.11 condition when catboost supports numpy 2.0
+lightgbm==4.6.0;
+catboost==1.2.8;
```

```suggestion
lightgbm==4.6.0
catboost==1.2.8
```
For alignment with the rest of the file